src/uct/api/uct.h

```diff
@@ -564,6 +569,22 @@ typedef struct uct_rkey_bundle {
 
 /**
  * @ingroup UCT_RESOURCE
+ * @brief Sockaddr with the callback and user's argument.
+ *
+ * This structure holds a sockaddr and a callback which is used when accepting
+ * connection messages while listening or connecting to this sockaddr.
+ */
+typedef struct uct_sockaddr_bundle {
```

Comment:
need a better name, e.g uct_sockaddr_params or uct_sockaddr_conn_params
Also need to improve documentation